cpp/cmake/thirdparty/get_nvcomp.cmake

```diff
@@ -16,11 +16,7 @@
 function(find_and_configure_nvcomp)
 
   include(${rapids-cmake-dir}/cpm/nvcomp.cmake)
-  rapids_cpm_nvcomp(
-    BUILD_EXPORT_SET cudf-exports
-    INSTALL_EXPORT_SET cudf-exports
-    USE_PROPRIETARY_BINARY ${CUDF_USE_PROPRIETARY_NVCOMP}
-  )
+  rapids_cpm_nvcomp(USE_PROPRIETARY_BINARY ${CUDF_USE_PROPRIETARY_NVCOMP})
```

Comment:
Not exporting this makes sense to me. But I suspect you'll also want to remove `nvcomp::nvcomp` from this manual install:

https://github.com/rapidsai/cudf/blob/9b2f892c5ec59605bfdc3a2abe4885176950589a/python/libcudf/CMakeLists.txt#L48-L52

Some context on what that is and why it's there today:

https://github.com/rapidsai/cudf/pull/15483#discussion_r1725988064

Reply:
I believe that the `cudf` part of this line was also unnecessary. I have removed the entire line.